pdf/sailfishofficepdfplugin.cpp

```diff
@@ -35,4 +36,8 @@ void SailfishOfficePDFPlugin::registerTypes(const char *uri)
     qmlRegisterType<PDFCanvas>(uri, 1, 0, "Canvas");
     qmlRegisterType<PDFLinkArea>(uri, 1, 0, "LinkArea");
     qmlRegisterType<PDFSelection>(uri, 1, 0, "Selection");
+    qmlRegisterType<PDFAnnotation>(uri, 1, 1, "Annotation");
+    qmlRegisterType<PDFTextAnnotation>(uri, 1, 1, "TextAnnotation");
+    qmlRegisterType<PDFCaretAnnotation>(uri, 1, 1, "CaretAnnotation");
+    qmlRegisterType<PDFHighlightAnnotation>(uri, 1, 1, "HighlightAnnotation");
```

Comment:
I think I'd just keep 1.0 for simplicity. Shouldn't be imported by anyone except this app. But doesn't really matter.